apps/prairielearn/src/pages/instructorQuestionSettings/instructorQuestionSettings.html.ts

```diff
@@ -236,6 +236,66 @@ export function InstructorQuestionSettings({
                   : ''}
               </table>
             </div>
+            <div class="form-group">
+              <label for="grading_method">Grading Method</label>
+              <select
+                class="form-select"
+                id="grading_method"
+                name="grading_method"
+                value="${resLocals.question.title}"
+                ${canEdit ? '' : 'disabled'}
+              >
+                <option
+                  value="Internal"
+                  ${resLocals.question.grading_method === 'Internal' ? 'selected' : ''}
+                >
+                  Internal
+                </option>
+                <option
+                  value="External"
+                  ${resLocals.question.grading_method === 'External' ? 'selected' : ''}
+                >
+                  External
+                </option>
+                <option
+                  value="Manual"
+                  ${resLocals.question.grading_method === 'Manual' ? 'selected' : ''}
+                >
+                  Manual
+                </option>
+              </select>
+              <small class="form-text text-muted">
+                The grading method used for this question.
+              </small>
+            </div>
+            <div class="form-group form-check">
+              <input
+                class="form-check-input"
+                type="checkbox"
+                id="single_variant"
+                name="single_variant"
+                ${canEdit ? '' : 'disabled'}
+                ${resLocals.question.single_variant ? 'checked' : ''}
+              />
+              <label class="form-check-label" for="single_variant">Single Variant</label>
+              <div class="form-text small text-muted">
+                If checked, the question is not randomized and only generates a single variant.
```

Comment:
This is not quite correct. The variant will still be randomized, students just won't be able to try multiple different variants. Here's my proposal (+/- Prettier):

```suggestion
                If enabled, students will only be able to try a single variant of this question on any given assessment.
```

Reply:
Ah, that makes more sense. Committed this in [5e60b9b](https://github.com/PrairieLearn/PrairieLearn/pull/11290/commits/5e60b9b96e63a404e2c245a3ce7da9be270772da).